helm/managed/logging/loki-stack/README.md

```diff
@@ -78,4 +78,57 @@ The default Loki-Stack chart will not declare permanent storage and therefore lo
 loki:
   persistence:
     enabled: true
-```
\ No newline at end of file
+```
+
+## Configure HPCC logAccess
+The logAccess feature allows HPCC to query and package relevant logs for various features such as ZAP report, WorkUnit helper logs, ECLWatch log viewer, etc.
+
+### Provide target Grafana/Loki access information 
+
+HPCC logAccess requires access to the Grafana username/password. Those values must be provided via a secure secret object.
+
+The secret is expected to be in the 'esp' category, and be named 'grafana-logaccess'. The following key-value pairs are required (key names must be spelled exactly as shown here)
+
+    grafana-username - This should contain the Grafana username
```

Comment:
curious why this is not just "username" and "password" since the secret name is already specific to grafana.

Reply:
agreed